src/adapters/PeopleAdapter.js

```diff
@@ -2,6 +2,21 @@ import {throwError} from 'rxjs';
 
 import WebexAdapter from './WebexAdapter';
 
+// Person Status Enum
```

Comment:
I really like this enum addition. Could you document it with an [enum](https://jsdoc.app/tags-enum.html) jsdoc?

Reply:
Also, let's get rid of the `PersonStatus` jsdoc typedef and just do something like:

```js
* @property {null|string}   status      The presence status of the user. @see PersonStatus enum
```